### ISSUE TITLE
thank and suggest

### ISSUE THREAD

**Issue by arliweng** [ianka/xfs_undelete#37]
thank u for great tool, two suggest,
1, i change the filename by ctime to mtime, unlike ext4 have directory name, so mtime is only mark of what it's..
hope an option for use the ctime or mtime as file name. or restore the file's mtime(in code current not)
2,  i put an size filter, many not .c but named .c file recover more 10G.. so very slow. hope an option for that, just `$loffset*$::blocksize` maybe.
i change the code but i don't known about tclsh, i just sample copy many if todo that, so i don't pull the request, just suggests.

**Comment by ianka** [comment]
You are right, using mtime instead of ctime for the filenames makes more sense. That code is from the original version which did not consider mtime and thus, did not retrieve it from the inode.

About the size filter, that's only for sorting out bogus files, right? Okay, it's easy to sort out all 10GB files as most likely bogus but it would be more useful to be able to sort out bogus 100 MB files as well. For the latter, it had to guess the file type earlier in the process. I have to think about that first.

**Comment by ianka** [comment]
I thought about it again, and I wonder if using mtime for the filename is really better. Usually people would know when they accidentally deleted the file (ctime) rather than when it was last updated (mtime).
